### DISCUSSION TITLE
CRAB NIPPLE - SVELATO IL MISTERO!

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#58] [Generale]
Ho scoperto cosa significano veramente quelle frasi con "crab" dette dai barboni.

Avevo letto che erano 2 barboni che facevano sesso e mi son fatto trascinare e convincere 
che la mia miglioria alla traduzione fosse quella corretta. (non indagando ulteriormente).

In due che facevano sesso avrebbe avuto anche un senso, ma io che fanno sesso proprio non li ho visti.

Avevo scritto da qualche parte, che se vedevo meglio i contesti, avrei potuto essere più preciso. Infatti, leggete e capirete.

Ma giocando il tutorial, sono alle prese con il primo barbone giù nelle scale a cui devo succhiare il sangue.
Lui è da solo e comincia a dire tutte quelle frasi con "crab" a ripetizione.

Già avevo dei dubbi su quella frase e mi sono insospettito, vedendo che il barbone è da solo e parla da solo.
Ma anche se sono in due o tre, la tiritera è sempre quella, non si toccano tra loro!

Ho letto che si sono fatti molti giri sui vari forum di traduzione per quella frase, ma nessuno ci ha capito una mazza!

Avevo detto che non sono un traduttore di Inglese, ma solo esperto di altre lingue di molto più ostiche dell'inglese.

In tutti questi anni nessuno ci era arrivato, dovevate apettare me per farlo...e scusate se è poco

Quindi la frase originale :
"There that crab nipple 'n this'n's got crab nipple - whole lotta crab nipple."

Tradotta da me in questo modo :
Ecco una strizzata a quel capezzolo e anche a questo una bella strizzata - un sacco di strizzate ai capezzoli."

Analizzando il contesto, in modo approfondito, mi sono accorto che non andava bene pur essendo apparentemente corretta e che stavano dicendo proprio tutt'altro!

Allora cerco...

Come ho visto le definizioni date, e sapendo il contesto, ho capito subito tutto!

Ma la ricerca non si è fermata lì, ho continuato, ed ecco tutte le definizione di "crab" che potrebbero essere consone al contesto.

crab : piattola

crab : brontolone 

crab about : lamentarsi di

crabs : (malattia venerea) pediculosi 

L’acronimo C.R.A.B. letteralmente significa ‘granchio’ ed è un acronimo inglese usato per definire la sintomaticità del mieloma: 
ipercalcemia, insufficienza renale, anemia e lesioni ossee. Quando si è in presenza di C.R.A.B., 
il paziente con mieloma multiplo deve essere sottoposto a trattamento.

dopo aver letto questo, cerco ancora :

In giro ho trovato anche questa frase in una singola risposta a un commento a cui nessuno ha replicato, inerente proprio alle frasi dei barboni del gioco :
Non solo noi, ma anche quelli di lingua anglofona sono stati in difficoltà a capire quelle frasi e non sono solo gli italianì in giro a domandare nei forum.
ma nessuno ne è venuto a capo.

Quindi in quel forum qualcuno chiedeva di "crab" etc...e qualcun'altro risponde così :
 
((("Crab" is slang for cancer so he's probably complaining about his tumors.)))

((("Crab" è un gergo per il cancro, quindi probabilmente si sta lamentando dei suoi tumori.)))

-----------------------------------------------------------------

Ok, ma dopo tutto questo, io mi son fatto un idea precisa. ( chi non si lava spesso, potrebbe avere le piattole...)

Io per "crab", ho scelto la parola "Piattola" (perchè analizzando le frasi, mi sembra la più adatta se non quella proprio perfetta per il contesto!

MISTERO SVELATO SOLO QUI DA NOI! 

"Hmm mmm mmm, cram met time crab nipple meh."
"Hmm mmm mmm, è tempo di strizzare le piattole mah."
o anche:
"Hmm mmm mmm, spassiamocela strizzando le piattole mah."

"Hey, crab nipple this 'n crab nipple that, 'n..."
"Hei, strizziamo questa piattola e strizziamo quest'altra, e..."

"There that crab nipple 'n this'n's got crab nipple - whole lotta crab nipple."
"Ecco una strizzata a quella piattola e anche a questa una bella strizzata - un sacco di strizzate alle piattole."

"Hmm, crab nipple hmm."
"Hmm, strizzo la piattola hmm."

"Take oh to the crab nipple, oh d'hey."
"E prendi una strizzata piattola e Muori."  ( "oh d'hey" significa "e muori." )

----------------------------------------------------------

Allora vi è piaciuta la favola del "granchio piattola"?

**Comment by crazyLillo** [comment]
mi ero scordato di dire che "nipple" in questo caso sta per "protuberanza" o "testa" (della piattola).
strizzare (schiacciare) la testa alla piattola (altrimenti non muore, come la zecca...)

**Comment by tesivo** [comment]
Caspita, colpo di scena! Davvero interessante! Grande crazyLillo!

Prima di applicare la traduzione:
della prima frase, hai proposto due traduzioni:
> "Hmm mmm mmm, cram met time crab nipple meh."
> "Hmm mmm mmm, è tempo di strizzare le piattole mah."
> o anche:
> "Hmm mmm mmm, spassiamocela strizzando le piattole mah."

Secondo me è migliore la prima. Tu che dici? Qualcun altro vuole dire la sua?

**Reply by NarakuITA** [reply]
Ottimo lavoro. La prima è la migliore!

**Reply by crazyLillo** [reply]
bhe! la prima e decisamente più consona. per quello che l'ho proposta per prima.

Dite la verità, non vi aspettavate questo retroscena.
Sono andato in giro su google, e ho visto che nessuno in Italia era riuscito  finora a dare la risposta giusta. Nessuno!
Persino i giocatori e presunti esperti di lingua anglofona non capivano che cazzo stessero dicendo questi poveri barboni.
Un presunto esperto (Inglese) su Steam, alla domanda sui barboni, ha risposto che forse intendevano scrivere "Ripple" cioè un vino di scarssissima qualità (ma noi adesso sappiamo che anche lui si è sbagliato di grosso). 

Non me ne capacito che dovevo arrivare io per svelare il mistero, non era poi così intraducibile la frase, avevo il contesto sotto gli occhi. Un buon vocabolario può fare la differenza se solo si ha voglia di farlo.
